### PR TITLE
Adding Coding Standards to Project

### PR DESCRIPTION
**Pull request by cbiddle3**
Fixes #issue_number

**What was changed?**

*Here, describe what part of the application you changed (e.g. login page, database, etc.). If possible, mention what specific components were added, removed, or modified.*

**Why was it changed?**

*Here, describe the issue that you are fixing with this code, and why your code fixes it.*

**How was it changed?**

*Here, get into detail about what files you modified, and talk about the most important lines in regards to fixing the issue.*

**Screenshots that show the changes (if applicable):**

### CODE REVIEW

File: CODING_STANDARDS.MD [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+### General Best Practices
+- Make sure to add unit test cases when adding new functionality and ensure added code is written in a testable manner.
+- All functions should have a description of their purpose and details of the parameters.
+- Soft limit of 100 characters per line.
+- Each issue should have one pull request unless the changes are only 1-3 lines.
+
+### Python Best Practices
+- Use snake-case for multi-word classes, variables, and functions.
+
+### JavaScript Best Practices
```
Comment (kungfuchicken) [inline CODING_STANDARDS.MD:12]: are these team prefs, or based on any standard styles? just curious.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+- Use the project's testing framework and conventions.
+- Write tests that cover the various scenarios and edge cases related to your code changes.
+- Ensure that the tests pass successfully before submitting a pull request.
+
+## Contacting the Team
+
+If you have any questions, concerns, or need further assistance, you can reach out to the project team through the following channel:
+
+[Slack channel](https://join.slack.com/t/oswslu/shared_invite/zt-24f0qhjbo-NkSfQ4LOg5wXxBdxP4vzfA)
```
Comment (kungfuchicken) [inline CONTRIBUTING.md:130]: This is the generic link to join our Slack instance. You should be able to create a custom link that puts people directly in #project_shelter_volunteers, or you can link to this to join slack, and refer to joining the project channel.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+If you have an idea for a new feature or enhancement, we encourage you to propose it to the project. Here's how you can do it:
+
+1. Check the existing issues in the [issue tracker](https://github.com/oss-slu/shelter_volunteers/issues) to see if it already exists. 
+2. Open a new issue [here]](https://github.com/oss-slu/shelter_volunteers/issues/new?assignees=&labels=&projects=&template=feature_request.md&title=) and follow the template.
+2. Provide a clear and descriptive title for the feature proposal.
+3. Explain the motivation and benefits of the proposed feature.
+4. Describe the proposed solution or implementation, if you have one in mind.
+5. Be open to discussions and feedback from the project maintainers and the community.
+
+## Contributing to Pull Requests
```
Comment (kungfuchicken) [inline CONTRIBUTING.md:41]: These are all great suggestions. You might want to link somewhere in this latter portion of the document to the program's "Community Partners" page (https://oss-slu.github.io/docs/about/community). Any suggestions for that page are welcome, too.

**Review by kungfuchicken — CHANGES_REQUESTED** [review]
Thank you! This looks generally great. Just a few questions clarify and notes to tidy up details. The only thing that must change is the Slack link.

File: CODING_STANDARDS.MD [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# Coding Standards 
+
+### General Best Practices
+- Make sure to add unit test cases when adding new functionality and ensure added code is written in a testable manner.
+- All functions should have a description of their purpose and details of the parameters.
+- Soft limit of 100 characters per line.
+- Each issue should have one pull request unless the changes are only 1-3 lines.
```
[the remark this answers is not in the document]
Comment (SiriChandanaGarimella) [reply CODING_STANDARDS.MD:7]: I meant a separate pull request is not needed if the changes are only 1-3 lines and can be part of another issue. It may be acceptable to include the code changes in an existing relevant pull request instead of creating a new one. Please let me know if it has to be updated.

File: CODING_STANDARDS.MD [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+### Python Best Practices
+- Use snake-case for multi-word classes, variables, and functions.
```
[the remark this answers is not in the document]
Comment (SiriChandanaGarimella) [reply CODING_STANDARDS.MD:10]: Yes, we gave this as per PEP variable name standard (https://peps.python.org/pep-0008/#function-and-variable-names) and also as per our current project standards.

File: CODING_STANDARDS.MD (hunk 1 above)
Comment (SiriChandanaGarimella) [reply CODING_STANDARDS.MD:12]: For Python, we gave this as per the PEP variable name standard (https://peps.python.org/pep-0008/#function-and-variable-names) and for react js, we referred to a few sites for clean coding and coding standards. This is also as per current project standards.

File: CONTRIBUTING.md (hunk 3 above)
Comment (SiriChandanaGarimella) [reply CONTRIBUTING.md:41]: I have made the changes. Could you please review them?

**Review by kungfuchicken — DISMISSED** [review]
lgtm;
